### ISSUE TITLE
cargo deb cross-compiling to `--target=i686-unknown-linux-gnu` fails

### ISSUE THREAD

**Issue by getreu** [mmstick/cargo-deb#119]
Cross-compiling with
```
$ cargo deb --target=i686-unknown-linux-gnu
```
fails:
```
    Finished release [optimized] target(s) in 0.01s
thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: CommandError("dpkg-query (get package version)", "libc6-i386:i386", [100, 112, 107, 103, 45, 113, 117, 101, 114, 121, 58, 32, 110, 111, 32, 112, 97, 99, 107, 97, 103, 101, 115, 32, 102, 111, 117, 110, 100, 32, 109, 97, 116, 99, 104, 105, 110, 103, 32, 108, 105, 98, 99, 54, 45, 105, 51, 56, 54, 58, 105, 51, 56, 54, 10])', src/libcore/result.rs:1165:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace.
thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: CommandError("dpkg-query (get package version)", "lib32gcc1:i386", [100, 112, 107, 103, 45, 113, 117, 101, 114, 121, 58, 32, 110, 111, 32, 112, 97, 99, 107, 97, 103, 101, 115, 32, 102, 111, 117, 110, 100, 32, 109, 97, 116, 99, 104, 105, 110, 103, 32, 108, 105, 98, 51, 50, 103, 99, 99, 49, 58, 105, 51, 56, 54, 10])', src/libcore/result.rs:1165:5
```

it seems the packages `libc6-i386:386` and `lib32gcc1:386`are missing. In Debian >= 10 they exist only as amd64 binaries.

My dependencies are most probably satisfied, because cross-compiling with `cargo build --target i686-unknown-linux-gnu --release` works without problems.

**Comment by kornelski** [comment]
Are you using the latest version? I'm surprised that you're getting a panic instead of a nicer message. Can you set `RUST_BACKTRACE=1` to help hunt this down>

**Comment by kornelski** [comment]
OK, found it.

**Comment by getreu** [comment]
I suppose the fix is this commit: b60ef2640e645d9689ba30b232b1b332e572a7b7 ?
Thank you for your fast reply.
Will it shipped with the next release?
